persist/src/main/resources/db/migration/V6_1__Neighborhood.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE IF NOT EXISTS neighborhood (
+    id INT PRIMARY KEY,
+    Name VARCHAR(36) NOT NULL,
+    SqMiles DOUBLE PRECISION,
+    Latitude DOUBLE PRECISION,
+    Longiude DOUBLE PRECISION
```

Comment:
```suggestion
    name VARCHAR(36) NOT NULL,
    sqMiles DOUBLE PRECISION,
    latitude DOUBLE PRECISION,
    longiude DOUBLE PRECISION
```
Prefer lowercase column names :)

Reply:
Also, sqMiles could be squareMiles, fine with either 🤷